### ISSUE TITLE
Create export to Terra "ready" component

### ISSUE THREAD

**Issue by MillenniumFalconMechanic** [clevercanary/data-browser#464]
#### Need
Once the backend has indicated the Terra export is ready, we need to display a "ready" page. See [mocks](https://www.figma.com/file/i4uZsYkKAML9ivi2TIo6ug/explore-dashboard?node-id=1012%3A19842). We want to encapsulate state this in a new component.

#### Approach
* Create an `ExportToTerraReady` component in /app/components/Export/ExportToTerra`.
* For now, "hard code" this component. We will revisit converting this component to be configurable in a separate ticket, if necessary.
* The component takes a single prop, `terraUrl`, which is a string.
* Re-use the existing `BatchNormalizationWarning` component.
* On click of "Open Terra", open a new tab at the `terraUrl` location.

#### Definition of Done
* Component displays text as per mocks.
* Component displays batch normalization warning.
* SB updated.
